pkg/operator/ceph/cr_manager.go

```diff
@@ -43,7 +46,16 @@ var (
 	}
 )
 
-func (o *Operator) startManager(namespaceToWatch string, context context.Context, mgrErrorCh chan error) {
+var (
+	// Default directory where TLS certs are stored - controller runtime's default
+	certDir = filepath.Join(os.TempDir(), "k8s-webhook-server", "serving-certs")
```

Comment:
Why use a temp dir instead of something like `/etc/webhook-certs`?

Reply:
Honestly, it does not really matter, everything disappears and gets recreated if the op container restarts

Reply:
It would just seem simpler to use a path more consistent with the other operator volumes if we left it under /etc